exporter/metric/metric_test.go

```diff
@@ -118,3 +118,159 @@ func TestRecordToMpb(t *testing.T) {
 		t.Errorf("%v", aggError)
 	}
 }
+
+
+func TestResourceToMonitoredResourcepb(t *testing.T) {
+	testCases := []struct {
+		resource *resource.Resource
+		expectedType string
+		expectedLabels map[string]string
+	}{
+		// k8s_container
+		{
+			resource.New(
+				kv.String("cloud.provider", "gcp"),
+				kv.String("cloud.zone", "us-central1-a"),
+				kv.String("k8s.cluster.name", "opentelemetry-cluster"),
+				kv.String("k8s.namespace.name", "default"),
+				kv.String("k8s.pod.name", "opentelemetry-pod-autoconf"),
+				kv.String("container.name", "opentelemetry"),		
+			), 
+			"k8s_container", 
+			map[string]string{
+				"project_id":     "",
+				"location": "us-central1-a",
+				"cluster_name": "opentelemetry-cluster",
+				"namespace_name": "default",
+				"pod_name": "opentelemetry-pod-autoconf",
+				"container_name": "opentelemetry",
+			},
+	    },
```

Comment:
Character spacing is a bit wonky in this file. I think you are using an editor that isn't consistently converting to tabs for you. Can you please fix this?

Also we need to set up a linter to automatically detect these kinds of things. I've added a ticket to track that here: https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/issues/59